XenAdmin/MainWindow.cs

```diff
@@ -2713,7 +2714,15 @@ private void SetClientUpdateAlert()
         {
             updateAlert = Updates.ClientUpdateAlerts.FirstOrDefault();
             if (updateAlert != null)
+            {

```

Comment:
We currently pull the url from the XCupdates.xml file so we would need to fetch that before we can download anything. 
So this is to cover the event that update checks are not automatic, so the menu item would need to trigger a fetch of the updates file and pull the source url. It would default to "Download latest source" instead of having the version number auto populated, I think this is doable.

You mentioning this made me notice a bug that if the auto check for updates is disabled the menu item under help is visible without a version number so I'll fix that to be by default invisible.